languages/elixir/exercises/concept/numbers/.docs/instructions.md

```diff
@@ -0,0 +1,44 @@
+In this exercise you'll be writing code to help a freelancer communicate with a project manager by providing a few utilities to quickly calculate daily and
+monthly rates, optionally with a given discount.
+
+We first establish a few rules between the freelancer and the project manager:
+
+- The daily rate is 8 times the hourly rate.
+- A month has 22 billable days.
+
+The freelancer is offering to apply a discount if the project manager chooses to let the freelancer bill per month, which can come in handy if there is a certain budget the project manager has to work with.
+
+Discounts are modeled as fractional numbers representing percentage, for example `25.0` (25%).
```

Comment:
This works and I don't think it needs to be changed, but something to think about.

Since we aren't using the string form with a trailing `%` sign, maybe it would be more correct to use a decimal form to represent the percentage.  e.g. `25% == 0.25`, `25.5% == 0.255`

Reply:
I considered both ways and I don't really have strong preference either way. The only reason I chose to go with `25.5` for 25% is that it adds one more tiny step that the student needs to do (`/ 100`) and I prefer to havee it since I already removed some complexity from the target exercise by not dealing with strings.

Reply:
That sounds like a good enough reason to me. :+1: